npm-fund: add page (#8495)
tldr-pages/tldr: pages/common/npm-fund.md

# npm fund

> > Retrive funding information.

Reviewer: ```suggestion
> Retrieve funding information from packages.
```

> > More information: <https://docs.npmjs.com/cli/v7/commands/npm-fund>.

Reviewer 2: ```suggestion
> More information: <https://docs.npmjs.com/cli/v8/commands/npm-fund>.
```

> - List information on funding dependencies of the current project:

Reviewer: ```suggestion
- List dependencies with funding URL for the project in the current directory:
```

`npm fund`

> - Fund a specific package:

Reviewer: ```suggestion
- Open the funding URL for a specific package in the default web browser:
```

`npm fund {{package}}`

> - Fund dependencies by filtering a specific [w]orkspace in the current project:

Reviewer: ```suggestion
- List dependencies with a founding URL for a specific [w]orkspace for the project in the current directory:
```
